packages/angular-ivy/package.json

```diff
@@ -57,7 +57,7 @@
     "lint": "run-s lint:prettier lint:eslint",
     "lint:eslint": "eslint . --format stylish",
     "lint:prettier": "prettier --check \"{src,test,scripts}/**/**.ts\"",
-    "yalc:publish": "ts-node ../../scripts/prepack.ts && yalc publish build --push"
+    "yalc:publish": "yalc publish build --push"
```

Comment:
Just noticed while testing with yalc that we don't need to call the prepack script here